app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -194,6 +194,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
        $scope.emsCommonModel.emstype == "openstack" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "scvmm" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "openstack_infra" && $scope.emsCommonModel.default_hostname ||
+       $scope.emsCommonModel.emstype == "nuage_network"  && $scope.emsCommonModel.default_hostname ||
```

Comment:
@Lokesh-Jain 
Please take a look at how the Endpoints portion of the form is implemented for other providers like `ems_container` or `ems_middleware` here:
https://github.com/ManageIQ/manageiq/blob/master/app/views/layouts/angular/_multi_auth_credentials.html.haml#L36
And follow those leads to implement it for `ems_network`.
Typically, the `endppoints` would comprise of these fields:`Hostname`, `API Port` and `Security Protocol`. 
You can hide the fields that you do not want displayed by passing the parameters like `:apiport_hide => true` or ` :security_protocol_hide => true` to the endpoints partial.

Also make sure you specify `ems_network` here https://github.com/ManageIQ/manageiq/blob/master/app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml#L1

If things are still unclear, I can send you a patch file that would have the above changes.
Thanks.